proto/v1/pagination.proto

```diff
@@ -17,21 +17,23 @@ option go_package = "./proto";
 package datacommons.v1;
 
 
-message ImportGroupCursor {
+// Cursor represents the position of a paginated Bigtable cache data for an
+// import group.
+message Cursor {
   // The index of the import group, starts from 0.
-  int32 import_group = 1;
+  int32 ig = 1;
   // The index of the page, starts from 0.
   int32 page = 2;
   // The position of the next read item in the current page, starts from 0.
   int32 item = 3;
 }
 
-message ImportGroupCursorGroup {
-  // Entity DCID or other information that identifies the ImportGroupCursorGroup.
+message CursorGroup {
+  // Entity DCID or other information that identifies the CursorGroup.
   string key = 1;
-  repeated ImportGroupCursor cursors = 2;
+  repeated Cursor cursors = 2;
 }
 
 message PaginationInfo {
-  repeated ImportGroupCursorGroup cursor_groups = 1;
+  repeated CursorGroup cursor_groups = 1;
```

Comment:
In what context is this repeated?  (It seems the impl expects a single entry).  Please clarify in comments.

Reply:
Each request entity corresponds to one cursor group. Bulk APIs would have more than one cursor group.

Reply:
Got it.